bcgov/HOUS-permit-portal
# Feature/bphh 220

## Description

This is an update so that permit applications are rendered from the backend, and currently the seeding sets up an initial render view of the permit application digitally.

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

[BPHH-220](https://hous-bssb.atlassian.net/browse/BPHH-220)
[BPHH-237](https://hous-bssb.atlassian.net/browse/BPHH-237) - sets up some basic pathing none of the implementation

## Steps to QA

- Ensure you've seeded the base requirement template (RequirementsFromXlsxSeeder.seed)
- Login as a submitter
- Click permit applications, if there is already a permit application you should see a rendered version of the permit application questions

## UI Accessibility Checklist

This HAS NOT been accessibility checked as it's a work in progress to give a base rendition of what has been seeded.

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?
